src/second-iter-mut.md

```diff
@@ -201,7 +201,7 @@ this works:
 
 * We `take` the `Option<&mut>` so we have exclusive access to the mutable
   reference. No need to worry about someone looking at it again.
-* Rust understands that it's ok to shard a mutable reference into the subfields
+* Rust understands that it's ok to share a mutable reference into the subfields
```

Comment:
I defer to the author but I think this may be correct. "To shard" can mean "to split up" and in this case we're splitting up a mutable reference into mutable references to each field

Reply:
Oh, makes sense! I didn't know it 😅

Reply:
No worries! I've only really heard of it in terms of databases. If you're interested in that kind of thing, [this Wikipedia article](https://en.wikipedia.org/wiki/Shard_(database_architecture)) is probably way more useful than I could ever be